app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+
+  <nav class="header">
+  <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+    <li>
+    <a href="/"><i class="fa fa-navicon"></i></a>
```

Comment:
- [ ] I recommend to try to make navbar as close to original design as possible, for example, you can descrease the font weight, size and center the elements:

Your navbar  | Behance navbar
------------- | -------------
<img width="382" alt="Screenshot 2022-05-06 at 23 20 17" src="https://user-images.githubusercontent.com/77038610/167203833-0d9c1524-e1f5-439b-9eb2-7a2fd48b6ac0.png"> | <img width="229" alt="Screenshot 2022-05-06 at 23 20 48" src="https://user-images.githubusercontent.com/77038610/167203924-7439ac55-baea-479d-937c-fafa6e2300ec.png">